pages/platform/ai/notebook_tuto_marine_mammal_sounds_classification/guide.en-gb.md

```diff
@@ -0,0 +1,87 @@
+- An AI Notebooks project created inside a [Public Cloud project](https://www.ovhcloud.com/en-gb/public-cloud/) in your OVHcloud account;
+- A user for AI Notebooks
+- Your own dataset
+
+## Instructions
+
+### Uploading your dataset on Public Cloud Storage
+
+If you want to upload it from the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB), go to the Object Storage section and create a new object container by clicking `Object Storage`{.action} > `Create an object container`{.action}.
+
+![image](images/new-object-container.png){.thumbnail}
+
+If you want to run it with the CLI, just follow this [this guide](https://docs.ovh.com/gb/en/publiccloud/ai/cli/access-object-storage-data/). You have to choose the region, the name of your container and the path where your data is located and use the following command:
+
+```bash
+ovhai data upload <region> <container> <paths>
+```
+
+> [!primary]
+>
+> This tutorial has been realized with the [Best of Watkins Marine Mammal Sound Database](https://cis.whoi.edu/science/B/whalesounds/index.cfm). If you don't have your own dataset, you can use it by downloading the dataset on [Kaggle](https://www.kaggle.com/shreyj1729/best-of-watkins-marine-mammal-sound-database/version/3).
+>
+
+### Launching and accessing Jupyter notebook with "One image to rule them all" framework
+
+> [!warning]
+>
+> Although this tutorial is based on the use of the **TensorFlow** image, we advise you to use this image: **One image to rule them all**. This will help you avoid errors when installing libraries such as Librosa or SoundFile, Python audio libraries.
+>
+
+You need to attach a volume if your data is in your OVHcloud Object Storage and you want to use it during your experiment. For more information on data, volumes and permissions, see [our guide on data](https://docs.ovh.com/gb/en/publiccloud/ai/cli/access-object-storage-data/).
+
+If you want to launch it from the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB), just follow this [guide][OVH Notebooks].
+
+`Choose the same region as your object container` > `"One image to rule them all" framework` > `Attach Object Storage containers (the one that contains your dataset)`
+
+If you want to launch it with the CLI, choose the [volume][OVH JOB cli] you want to attach and the number of GPUs (`<nb-gpus>`) to use on your notebook and use the following command:
+
+```bash
+ovhai notebook run one-for-all jupyterlab \
+  --name <notebook-name> \
+  --gpu <nb-gpus>
+  --volume <container@region/prefix:mount_path:permission>
+```
+
+You can then reach your notebook’s URL once the notebook is Running.
+
+### Cloning the GitHub repository
+
+The GitHub repository containing all examples for OVHcloud AI TRAINING is available [here][GITHUB OVHAI Examples].
```

Comment:
Missing link. Please edit it and replace [Text for link][URL] with [Text for link](URL).